src/app/_shared-components/Profile/Overview/Tips/Tips.tsx

```
@@ -0,0 +1,181 @@
+								className={cn(classes.tab, 'h-7', activeTab === tab ? classes.activeTab : classes.inactiveTab)}
+							>
+								{t(tab)}
+							</Button>
+						))}
+					</div>
+				</div>
+			</div>
+			{isLoading ? (
+				<div className='flex h-full flex-col items-center justify-center gap-1'>
+					{new Array(3).fill(0).map(() => (
+						<Skeleton className='h-12 w-full' />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)